components/policy/resources/templates/policy_definitions/BraveSoftware/BraveAIChatEnabled.yaml

```diff
@@ -0,0 +1,39 @@
+caption: Disable AI Chat
+default: null
+desc: |-
+  Disable AI Chat in Brave.
+
+  The smart AI assistant built right into your browser. Ask questions, summarize pages, create new content, and more. Privately.
```

Comment:
cc: @mattmcalister on the wording here! I grabbed it from https://brave.com/leo/. This text will show in the Windows group policy editor. It looks like this when it's displayed:
![image](https://github.com/user-attachments/assets/ebb2f533-2961-4907-b8f3-b748f4d44e20)